src/subscriptions.jl

```diff
@@ -120,11 +132,13 @@ function open_subscription(fn::Function,
                 output_info(verbose) && println("Subscription $sub_id stopped by the stop function supplied")
                 break
             end
-            response = JSON3.read(data::Vector{UInt8}, GQLSubscriptionResponse{output_type})
+            # Dropping typeassert as this may be string
```

Comment:
Don't need the comment here - won't make sense when the type assert isn't there!
```suggestion
```